tests/test_utils.py

```diff
@@ -405,14 +405,8 @@ def test_convert_dict_to_env_variables(self):
             valid_env_items = convert_dict_to_env_variables(env)
         assert valid_env_items == ["ACCELERATE_DEBUG_MODE=1\n", "OTHER_ENV=2\n"]
 
-
-class TqdmTester(unittest.TestCase):
-    def test_tqdm(self):
-        expected = [0, 1, 2]
-        assert list(tqdm(range(3), main_process_only=True)) == expected
-        assert list(tqdm(main_process_only=True, iterable=range(3))) == expected
-
-        assert list(tqdm(range(3), main_process_only=False)) == expected
-        assert list(tqdm(main_process_only=False, iterable=range(3))) == expected
-
-        assert list(tqdm(range(3))) == expected
+    def test_tqdm_deprecation(self):
+        with pytest.warns(FutureWarning) as cm:
+            tqdm(True, range(3), disable=True)
+        assert "Passing `True` as the first argument to" in cm.pop().message.args[0]
+        tqdm(range(3), main_process_only=True, disable=True)
```

Comment:
Should this line also check that there is [no warning](https://docs.pytest.org/en/7.0.x/how-to/capture-warnings.html#additional-use-cases-of-warnings-in-tests)?

```python
with warnings.catch_warnings():
    warnings.simplefilter("error")
```